test/fixtures/responses/get-ledger-pre2014-with-partial.json

```diff
@@ -25,12 +25,7 @@
           }
         },
         "destination": {
-          "address": "rNNuQMuExCiEjeZ4h9JJnj5PSWypdMXDj4",
-          "amount": {
-            "currency": "USD",
-            "value": "10",
-            "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B"
-          }
```

Comment:
I don't like how we lose the destination currency/counterparty.
What do you think about keeping the amount and either omitting or renaming `"value"`?

Reply:
I think the destination currency/counterparty can still be seen in deliveredAmount, or if that's not present, then under balanceChanges. I'll test this out.

Reply:
I've also been considering just renaming amount to provisionalAmount or requestedAmount, but then there's still the risk that someone would misinterpret it.

Reply:
I mean, really, the problem is that people tend to assume that "success" means "it did what you think it did".

You'd think that people would learn that's now how computers work, but it keeps happening.

Reply:
Good point. I guess we can say that the transaction was applied to the ledger, but you have to parse the outcome to find out what it really did.